<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>beach_add</class>
 <widget class="QDialog" name="beach_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="shower_text">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>샤워실 개수</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="visitor_text">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_text">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="update_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>정보 등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>이름 중복확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>방문객 수</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>탈의실 개수</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dress_text">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="toilet_text">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>해수욕장 명</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>화장실 개수</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
